nailgun/entities.py

```diff
@@ -2774,6 +2778,17 @@ def __init__(self, server_config=None, **kwargs):
         self._meta = {'api_path': 'api/v2/ptables', 'server_modes': ('sat')}
         super(PartitionTable, self).__init__(server_config, **kwargs)
 
+    def update(self, fields=None):
+        """Fetch a complete set of attributes for this entity."""
```

Comment:
Is there a bugzilla bug stating what's wrong? If not, can we make one? And if so, can we link to it?

My goal is to make sure that every single work-around in NailGun is associated with a bug, even when that means filing trivial bugs or bugs against old versions of Satellite. I'm tracking progress in #105. Feel free to add this method to that list. ([Some](https://bugzilla.redhat.com/show_bug.cgi?id=1230865) [examples](https://bugzilla.redhat.com/show_bug.cgi?id=1234973) [of](https://bugzilla.redhat.com/show_bug.cgi?id=1237257) [bugs](https://bugzilla.redhat.com/show_bug.cgi?id=1235379))